dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1653,6 +1658,103 @@ public void pageWithSidebarShouldRender() throws Exception{
+        final Template template = new TemplateDataGen()
+                .host(site)
+                .drawedBody(templateLayout)
+                .theme(theme)
+                .nextPersisted();
+
+        final String pageName = "test1Page-"+System.currentTimeMillis();
+        final HTMLPageAsset page = createHtmlPageAsset(template, pageName, 1);
+
+        final Persona defaultPersona = APILocator.getPersonaAPI().getDefaultPersona();
+        final Persona notDefaultPersona = new PersonaDataGen().nextPersisted();
+
+        final Contentlet defaultPersonaContentlet = new ContentletDataGen(contentGenericType.id())
+                .languageId(1)
+                .folder(folder)
+                .host(site)
+                .setProperty("title", "content1")
+                .setProperty("body", defaultPersona.getKeyTag())
+                .nextPersisted();
+
+        final Contentlet notDefaultPersonaContentlet = new ContentletDataGen(contentGenericType.id())
+                .languageId(1)
+                .folder(folder)
+                .host(site)
+                .setProperty("title", "content1")
+                .setProperty("body", notDefaultPersona.getKeyTag())
+                .nextPersisted();
+
+        ContentletDataGen.publish(page);
+        ContentletDataGen.publish(defaultPersonaContentlet);
+        ContentletDataGen.publish(notDefaultPersonaContentlet);
+
+        new MultiTreeDataGen()
+                .setContainer(container)
+                .setPage(page)
+                .setContentlet(defaultPersonaContentlet)
+                .setInstanceID("1")
+                .nextPersisted();
+
+        new MultiTreeDataGen()
+                .setContainer(container)
+                .setPage(page)
+                .setContentlet(notDefaultPersonaContentlet)
+                .setPersona(notDefaultPersona)
+                .setInstanceID("1")
+                .nextPersisted();
+
+
+        //request page ENG version
+        HttpServletRequest mockRequest = new MockSessionRequest(
+                new MockAttributeRequest(
+                        new MockHttpRequest("localhost", "/").request()).request()
+            ).request();
+
+        when(mockRequest.getParameter("host_id")).thenReturn(site.getIdentifier());
+        mockRequest.setAttribute(WebKeys.HTMLPAGE_LANGUAGE, "1");
+        HttpServletRequestThreadLocal.INSTANCE.setRequest(mockRequest);
+        final HttpServletResponse mockResponse = mock(HttpServletResponse.class);
+
+        final String htmlWithDefaultPersona = APILocator.getHTMLPageAssetRenderedAPI().getPageHtml(
+                PageContextBuilder.builder()
+                        .setUser(systemUser)
+                        .setPageUri(page.getURI())
+                        .setPageMode(PageMode.LIVE)
+                        .build(),
+                mockRequest, mockResponse);
+        Assert.assertTrue(htmlWithDefaultPersona.contains("dot:persona"));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.assertTrue' due to existing static import 'org.junit.Assert.assertTrue'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6340183)